modules/development/pages/management-vm.adoc

```diff
@@ -0,0 +1,98 @@
+= Management VM
+:toc:
+:toclevels: 3
+
+WARNING: This page is a work in progress.
+
+. Copy the management VM image from `node-images` into the appropriate location.
```

Comment:
is this ran on a hypervisor?

`crucible vm start` does all of these steps.

Reply:
I have yet to use crucible and in my mind I think of crucible as something specific to our hypervisors for now. I'm trying to keep steps manual for development to define the how.